packages/apps/tools/src/components/Global/HoverTag/HoverTag.tsx

```diff
@@ -0,0 +1,49 @@
+import type { IMaskedValueProps } from '@kadena/react-ui';
+import { SystemIcon, Tooltip, maskValue } from '@kadena/react-ui';
+import React, { useRef } from 'react';
+import {
+  closeButtonClass,
+  tagClass,
+  tagStyles,
+  tooltipStyles,
+} from './HoverTag.css';
+
+export interface IHoverTagProps {
+  value: IMaskedValueProps['value'];
+}
+
+export const HoverTag = ({ value }: IHoverTagProps) => {
+  const [tooltipContent, setTooltipContent] = React.useState<string>(value);
+  const ref = useRef<HTMLDivElement>(null);
+  return (
+    <>
+      <span
+        data-testid="kda-hover-tag"
+        className={tagClass}
+        onMouseEnter={(e: React.MouseEvent<HTMLElement>) =>
+          Tooltip.handler(e, ref)
+        }
+        onMouseLeave={(e: React.MouseEvent<HTMLButtonElement>) =>
+          Tooltip.handler(e, ref)
+        }
+      >
+        <span className={tagStyles}>{maskValue(value)}</span>
```

Comment:
Do you think it might be better to have a `MaskedText` component rather than utility function since we don't expose any other utility functions? 

I'm thinking a `MaskedText` component that accepts a string with an optional disabled prop

Reply:
I can do that, but I do think I would need to use this utility function in other places. And also; there wouldn't be any harm in exposing this utils function, right?

Could we keep it as is for now and if I do encounter many use cases where I am just using this `maskValue` function to show as a text I will then refactor this and make a `MaskedText` component. Agreed?

Reply:
yeah, that works 👍

Reply:
Will do, thanks Eileen 👌